internal/io/embedio.go

```diff
@@ -0,0 +1,15 @@
+package io
```

Comment:
This is missing our license header

Reply:
Also, having a whole package for a single function looks like an overkill

Reply:
The io package already exists. The other files did not seem appropriate in this package, so I choose a new. If you wish a different approach, then please give a more detailed description of what you have in mind.